### PR TITLE
add open access tag lf ubtue/DatenProbleme#1554

### PR DESCRIPTION
**Pull request by socheres**

### CODE REVIEW

File: Central and Eastern European Online Library Journals.js [hunk 1]
```diff
@@ -0,0 +1,307 @@
+{
```
Comment (jriedl) [inline Central and Eastern European Online Library Journals.js:1]: Shouldn't our own translators bew prefixed with `ubtue_`?

File: Springer Link.js [hunk 2]
```diff
@@ -197,19 +197,15 @@ function complementItem(doc, item) {
 
 		item.abstractNote = abstract.trim();
 	} else {
-		let absSections = ZU.xpath(doc, '//*[(@id = "Abs2-content")]//p');
-		let sectionTitles = ZU.xpath(doc, '//*[(@id = "Abs2-content")]//*[contains(concat( " ", @class, " " ), concat( " ", "c-article__sub-heading", " " ))]');
-		let titleTextGerman = ZU.xpathText(doc, '//*[(@id = "Abs1-content")]//p');
+		let otherAbstracts = doc.querySelectorAll('#Abs2-section');
```
Comment (jriedl) [inline Springer Link.js:200]: How can we ensure that there are no regression problems, i.e. that the now removed code is not needed in another context?

File: Wiley Online Library.js [hunk 3]
```diff
@@ -262,15 +262,7 @@ function scrapeBibTeX(doc, url) {
...
-			// BibTeX throws the last names and first names together
-			// Therefore, we prefer creators names from EM (if available)
-			var authors = doc.querySelectorAll('meta[name="citation_author"]');
-			if (authors && authors.length>0) {
-				item.creators = [];
-				for (let i=0; i<authors.length; i++) {
-					item.creators.push(ZU.cleanAuthor(authors[i].content, 'author'));
-				}
-			}
+
```
Comment (jriedl) [inline Wiley Online Library.js:265]: How can we be sure that this is not needed in a different context?

File: Central and Eastern European Online Library Journals.js (hunk 1 above)
Comment (socheres) [reply Central and Eastern European Online Library Journals.js:1]: This is the official upstream version from zotero/translator

File: Springer Link.js (hunk 2 above)
Comment (socheres) [reply Springer Link.js:200]: i run the test for all 15 urls and with the previous version i got wrong abstract e.g. https://link.springer.com/article/10.1007/s00481-021-00606-5

File: Wiley Online Library.js (hunk 3 above)
Comment (socheres) [reply Wiley Online Library.js:265]: i have taken the change from the master file.

Comment (socheres) [reply Wiley Online Library.js:265]: see also changed test results.

File: Central and Eastern European Online Library Journals.js (hunk 1 above)
Comment (jriedl) [reply Central and Eastern European Online Library Journals.js:1]: Ok.

File: Springer Link.js (hunk 2 above)
Comment (jriedl) [reply Springer Link.js:200]: One then might change the order of evaluation, but ok then we go with it.

File: Wiley Online Library.js (hunk 3 above)
Comment (jriedl) [reply Wiley Online Library.js:265]: see above.

**Comment by jriedl** [comment]
In Kürze auf nu.

**Pull request merged**